PlannTer/Modules/PlantModule/PlantAddView.swift

```diff
@@ -184,3 +250,8 @@ private struct MiniButton : View {
     }
 }
 
+#Preview{
+    PlantAddView(room: RoomModel.exampleRoom)
```

Comment:
this crashes the preview - check `MainScreenView` for solution

Reply:
solution from main screen seems to have issues here, just skipping it altogether